src/reducers/index.js

```diff
@@ -19,8 +19,15 @@ const reducer = (state = getDefaultState(), action) => {
         roomId: action.roomId
       });
     case 'GET_ROOMS':
+      const rooms = action.roomIds.map((roomId) => {
+        return {roomId}
+      });
+      return Object.assign({}, state, {
+        rooms
+      });
+    case 'GET_PLAYERS':
```

Comment:
Maybe this action_type is also no longer used?

I think you just use only [`GET_ROOM_INFO`](https://github.com/duyoji/oni-on-cheese/pull/92/files#diff-cfba9c145ca5d713b626b9c58bb518a7R52) instead of `GET_ROOMS` and `GET_PLAYERS`